docs/topics/mtr-rn-known-issues-1-2-1.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * docs/release-notes-mtr/master.adoc
+
+:_content-type: REFERENCE
+[id="mtr-rn-known-issues-1-2-1_{context}"]
+
+= Known issues
+
+For a complete list of all known issues, see the list of link:https://issues.redhat.com/browse/WINDUP-4043?filter=12422643[MTR 1.2.1 known issues] in Jira.
```

Comment:
@PhilipCattanach , This filter displays bugs that were fixed in previous MTR versions and in status VERIFIED,
as well as bugs that are targeted to previous releases, but still in New / assigned 

Maybe we can add this filter not to include status = Verified,
or do you suggest another solution?

Reply:
Removed the `Verified` status, so you can see what it looks like